Update harness-platform-faqs.md (#6636)
harness/developer-hub: kb/platform/harness-platform-faqs.md

  -e MANAGER_HOST_AND_PORT=https://app.harness.io harness/delegate:yy.mm.verno
```

> ###  Does harness http step support edgegrid auth?

Reviewer: ```suggestion
###  Does the HTTP step support EdgeGrid authorization?
```

> You can pass Client token, Access token, Nonce as header for http step and it should work.

Reviewer: ```suggestion
You can pass the Client token, Access token, and Nonce as the header for the HTTP step for EdgeGrid authorization.
```
